concepts/collateral-breakdown.md

```diff
@@ -0,0 +1,19 @@
+# Collateral Breakdown
+
+Collateral assets held by the Maker Protocol or its SubDAOs can be broken up into the following categories.
+
+{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+## Decentralized collateral
+1. Ether: Examples include Ether, Liquid Staking Derivatives such as stETH or rETH, ETH/DAI liquidity pool tokens or [EtherDAI](etherdai.md).
+2. Miscellaneous: Examples include ERC-20 tokens such as UNI, LINK, etc. as well as liquidity pool tokens involving these assets.
+3. MKR: Note that MKR no longer functions as the backstop if the protocol is insolvent.
```

Comment:
Probably not worth including the 'no longer backstop' thing in here. I know I did in the summary you took this from, but really it's not strictly relevant to the breakdown of collateral types.

Reply:
I added some more detail and thought it best to keep the no backstop thing to provide context. Let me know if this version is okay